plugins/pip.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "pip",
+    "version": "0.0.1",
+    "match": "^python3[0-9]{1,2}Packages.pip",
+    "readme": "This plugin adds a script for automatically creating a virtual environment using `venv` for python3 projects, so you can install packages with pip as normal.\nTo activate the environment, run `source $VENV_DIR/bin/activate` or add it to the init_hook of your devbox.json\nTo change where your virtual environment is created, modify the $VENV_DIR environment variable in your init_hook",
+    "env": {
+        "VENV_DIR": "{{ .Virtenv }}/pip/.venv"
+    },
+    "create_files": {
+        "{{ .DevboxDir }}/venvShellHook.sh": "pip/venvShellHook.sh"
+    },
+    "shell": {
+        "init_hook": [
+            "sh {{ .DevboxDir }}/venvShellHook.sh"
```

Comment:
Is `sh` needed? I think you can just add `#!/bin/sh` to the file? If you put it into a `bin` directory it automatically gets run permissions https://github.com/jetpack-io/devbox/blob/landau/start-auto-forward-on-cloud/internal/plugin/pkgcfg.go#L89-L93

Reply:
So I could do something like: 

```json
    "shell": {
        "init_hook": [
            "{{ .DevboxDir }}/bin/venvShellHook.sh"
        ]
    }
```
?